### PR TITLE
importing heads parameters from the correct file on tutorial 1

### PR DESCRIPTION
**Pull request by piEsposito**
importing heads parameters from the correct file, otherwise it crashes. It was importing HeadParameters from **rl_coach.architectures.tensorflow_components.heads.head** which was making the tutorial to crash and the correct is **rl_coach.architectures.head_parameters**

### CODE REVIEW

**Comment by piEsposito** [comment]
Also I've reseted tensorflow default graph before creating new ones on all tutorial 0; otherwise it is not crashis because of the already existent variables

File: tutorials/.ipynb_checkpoints/0. Quick Start Guide-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,439 @@
+{
```
Comment (gal-leibovich) [inline tutorials/.ipynb_checkpoints/0. Quick Start Guide-checkpoint.ipynb]: Please remove the saved ipynb  checkpoint file, as the tutorial should be clean

Comment (piEsposito) [reply tutorials/.ipynb_checkpoints/0. Quick Start Guide-checkpoint.ipynb]: done: the ipynb checkpoint file is removed.

File: tutorials/0. Quick Start Guide.ipynb [hunk 2]
```diff
@@ -430,7 +431,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.4"
+   "version": "3.6.3"
```
Comment (gal-leibovich) [inline tutorials/0. Quick Start Guide.ipynb]: Why this change? if no real reason, I suggest changing back to 3.6.4

File: tutorials/0. Quick Start Guide.ipynb [hunk 3]
```diff
@@ -242,6 +244,9 @@
     "from rl_coach.graph_managers.graph_manager import SimpleSchedule\n",
     "from rl_coach.architectures.embedder_parameters import InputEmbedderParameters\n",
     "\n",
+    "#reseting tensorflow graph as we will use other network\n",
```
Comment (gal-leibovich) [inline tutorials/0. Quick Start Guide.ipynb]: Whitespace is missing after the comment's '#' sign. Also, please consider changing the comment to `Resetting tensorflow graph as the network has changed."

File: tutorials/0. Quick Start Guide.ipynb (hunk 2 above)
Comment (piEsposito) [reply tutorials/0. Quick Start Guide.ipynb]: Thank you for noticing it. Done.

File: tutorials/0. Quick Start Guide.ipynb (hunk 3 above)
Comment (piEsposito) [reply tutorials/0. Quick Start Guide.ipynb]: Also done. I did accept the suggestion. Much better now, thank you!
